apps/web/src/actions/evaluations/generateSuggestedEvaluations.ts

```diff
@@ -51,6 +51,7 @@ export const generateSuggestedEvaluationsAction = authProcedure
       env.COPILOT_EVALUATION_SUGGESTION_PROMPT_PATH,
       {
         stream: false,
+        versionUuid: 'da47b89d-2bde-4c6c-92ee-11a17241eb73', // TODO: remove
```

Comment:
will get removed once i've deployed to production this commit and all production users are using the new evaluation generator